### PR TITLE
chore: update banner content

### PR DESCRIPTION
**Pull request by paulatulis**
Screen Shot 2020-09-10 at 2 56 56 PM

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 04c899553b91d2aa667a67fc9aa263107587dcb4

https://deploy-preview-1595--stoplightio.netlify.app

File: netlify/settings.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 banners:
   - ends: 1609430400000
     markdown: >-
-      Interested in a live tour of Stoplight Platform 2.0? <u class="ml-1">[Register
-      here](https://us02web.zoom.us/webinar/register/WN_1Qe8hgoARmqX4tWgb88t0g)</u> to join us on September 17!
+      Go deep on API Design and explore the Stoplight Platform with Architect, Phil Sturgeon. Register for the
+      interactive webinar <u
+      class="ml-1">[here](https://us02web.zoom.us/webinar/register/WN_1Qe8hgoARmqX4tWgb88t0g)</u>.
```
Comment (texas-kristen) [inline netlify/settings.yaml:6]:
```suggestion
      Learn the ins-and-outs of the Stoplight Platform in our Stoplight 101 series. Register for the interactive workshop  <u
      class="ml-1">[here](https://us02web.zoom.us/webinar/register/WN_aimSFMlCQeCTNDnTiZieqQ)</u>.
```

**Review by texas-kristen — APPROVED** [review]

File: netlify/settings.yaml (hunk 1 above)
Comment (paulatulis) [reply netlify/settings.yaml:6]: @texas-kristen Just want to make sure before I take any action. Want me to go ahead and commit this suggestion and merge?

Comment (texas-kristen) [reply netlify/settings.yaml:6]: Yep, we're good to go

**Pull request merged**